src/bindgen/config.rs

```diff
@@ -248,6 +248,8 @@ pub struct ExportConfig {
     pub renaming_overrides_prefixing: bool,
     /// Name mangling character. Defaults to `_`
     pub mangle_separator: Option<String>,
+    /// Whether to capitalise primitives, e.g. if mangle_separator is empty
+    pub pascal_case_primitives: bool,
```

Comment:
We have another rename rules (see `RenameRule`) that you can use, it seems using that would be better for consistency, rather than special-casing pascal-case here.

Reply:
OK thanks, I was not aware of that, see my comment below.